scripts/sct_apply_transfo.py

```diff
@@ -142,6 +151,9 @@ def apply(self):
         verbose = self.verbose
         remove_temp_files = self.remove_temp_files
         crop_reference = self.crop  # if = 1, put 0 everywhere around warping field, if = 2, real crop
+        label=self.label
```

Comment:
space. Please apply PEP8 for all your modifs (there are other formatting issues below)